CC/Sounder/include/RadioUHD.h

```diff
@@ -0,0 +1,46 @@
+/*
+ Copyright (c) 2018-2019, Rice University
+ RENEW OPEN SOURCE LICENSE: http://renew-wireless.org/license
+
+*/
+
+#include <SoapySDR/Device.hpp>
+#include <SoapySDR/Errors.hpp>
+#include <uhd/usrp/multi_usrp.hpp>
+
+#include "config.h"
+
+#ifndef RADIO_UHD_H
+#define RADIO_UHD_H
+
+class RadioUHD {
+ private:
+  uhd::usrp::multi_usrp::sptr dev;
+  uhd::rx_streamer::sptr rxs;
+  uhd::tx_streamer::sptr txs;
+
+  void reset_DATA_clk_domain(void);
+  void dev_init(Config* _cfg, int ch, double rxgain, double txgain);
+  friend class ClientRadioSetUHD;
```

Comment:
It would be nice to get rid of this "friend class stuff"  but since it is in the current code.  We can leave it here for now